### DISCUSSION TITLE
776

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#948] [Comments]
# 776

요즘 픽사의 CEO 에드 캣멀이 쓴 "창의성을 지휘하라" 를 보고 있다. 여기를 보면 스타트업에 너무 적합한 이야기가 있다. "사람들은 효과가 검증된 것, 예컨대 과거에 통한 스토리, 방법, 전략에 안주하고 싶어한다. 새로 고안한 방법이 효과가 있다고 판명되면, 이 방법을 계속 사용한다. 조직은 이런식으로 학습한다. 조직은 성공을 거둬 성장할수록 기존 접근법에 집착하고, 점점 더 변화를 거부하게 된다." "안정적인 것을 추구한다고 안전해진다는 보장은 없다. 무작위성을 두려워하는 대신 인생에서 무작위성을 피할 수 없다는 사실을 인식하고, 무작위성을 긍정적으로 활용하는 선택을 내릴 수 있다고 믿는다. 예측할 수 없는 상황은 창의성의 산실이다." "픽사의 열 번째 장편 애니메이션 업(UP) 은 픽사의 작품 중..

https://jojoldu.tistory.com/776

**Comment by yoonjiyong** [comment]
안녕하세요~ 
스타트업 서비스 준비중인 사람 입니다.
혹시 개발 관련 컨설팅도 하시나요..?
초기 개발팀 필수 인원 구성에 대한 내용 입니다.
